### PR TITLE
ENH: Update CI for ITK v5.2rc02

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Review by phcerdan — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -44,6 +44,6 @@
     keywords='ITK InsightToolkit',
     url=r'https://itk.org/',
     install_requires=[
-        r'itk>=5.2rc1'
+        r'itk>=5.2rc2'
```
Comment (jhlegarreta) [inline setup.py:47]: Is `5.2rc2` this equivalent to `5.2rc02` (used in the workflow config file and correspondig to the ITK tag) / are `0`s dropped ?

**Review by jhlegarreta — COMMENTED** [review]
An in-line comment.

File: setup.py (hunk 1 above)
Comment (thewtex) [reply setup.py:47]: Yes, Python package name semantics do not have the `0`, but the ITK tag's do.

Comment (jhlegarreta) [reply setup.py:47]: :+1:

**Pull request merged**